js_modules/dagster-ui/packages/ui-components/src/theme/lightHighContrastThemeColors.tsx

```diff
@@ -0,0 +1,129 @@
+import {css} from 'styled-components';
+
+export const lightHighContrastThemeColors = css`
+  --browser-color-scheme: 'light';
```

Comment:
This should be `light`, without the quotes. (Should do the same for `lightThemeColors`)

```suggestion
  --browser-color-scheme: light;
```